### PR TITLE
NameRegistry, onboarding flow

### PR DESCRIPTION
**Pull request by dcposch**
@nalinbhardwaj

### CODE REVIEW

**Review by nalinbhardwaj — APPROVED** [review]
lgtm; might want to add an `owner` and override methods for them here so we can manual override if someone takes `vitalik` or something (at least while we're on testnet)

File: packages/daimo-api/src/model.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Address } from "viem";
+import { z } from "zod";
```
Comment (nalinbhardwaj) [inline packages/daimo-api/src/model.ts:2]: cool library; im seeing it for the first time

**Comment by dcposch** [comment]
ds

File: packages/daimo-api/gen/contract.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Generated by @wagmi/cli@1.1.0 on 6/3/2023 at 9:46:11 PM
+// Generated by @wagmi/cli@1.1.0 on 6/7/2023 at 3:04:12 AM
```
Comment (nalinbhardwaj) [inline packages/daimo-api/gen/contract.ts:1]: 👀that’s late haha🌃

**Pull request merged**